test/e2e/specs/editor/blocks/links.spec.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Links', () => {
+	test.use( {
+		linkControl: async ( { page }, use ) => {
+			await use( new LinkControl( { page } ) );
+		},
+	} );
+
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
+	} );
+
+	test( `can be created by selecting text and using keyboard shortcuts`, async ( {
+		page,
+		editor,
+		pageUtils,
+		linkControl,
+	} ) => {
+		// Create a block with some text.
+		await linkControl.clickBlockAppender();
+		await page.keyboard.type( 'This is Gutenberg' );
+
+		// Select some text.
+		await pageUtils.pressKeys( 'shiftAlt+ArrowLeft' );
+
+		// Press Cmd+K to insert a link.
+		await pageUtils.pressKeys( 'primary+K' );
+
+		// Type a URL.
+		await page.keyboard.type( 'https://wordpress.org/gutenberg' );
+
+		// Open settings.
+		await page.keyboard.press( 'Tab' );
+		await page.keyboard.press( 'Space' );
+
+		// Navigate to and toggle the "Open in new tab" checkbox.
+		await page.keyboard.press( 'Tab' );
+		await page.keyboard.press( 'Space' );
```

Comment:
Could we try to avoid relying on tab ordering and rather find these by accessible attributes.

Why? Because as we refactor Link Control the order of tabs may change (and that's ok). That may also happen in the future and we don't want all the tests to fail each time that happens.

Rather I'd love it if we had a _single_ tests which asserted on keyboard interactions and tab stops. Then if something changes we can have an explicit test alert us.